src/lib/vm.ts

```diff
@@ -323,4 +327,59 @@ export async function registerVM(program: Command) {
         throw err;
       }
     });
+
+  vm.command("replace")
+    .description("Replace a virtual machine")
+    .requiredOption("-i, --id <id>", "ID of the VM to replace")
+    .action(async (options) => {
+      // Replace is a destructive action - get confirmation
+      const replaceConfirmed = await confirm({
+        message:
+          `Are you sure you want to replace VM instance ${options.id}? (You cannot undo this action)`,
+        default: false,
+      });
+      if (!replaceConfirmed) {
+        process.exit(0);
+      }
+
+      const loadingSpinner = ora(`Replacing VM ${options.id}...`).start();
```

Comment:
```suggestion
      const loadingSpinner = ora(`Replacing VM ${options.id}`).start();
```

No need ellipsis, the spinner conveys loading state. Ellipsis added more information but not extra signal